src/legacy/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -349,7 +349,7 @@ function discoverController(
 
     return [
       newSearch,
-      ...uiCapabilities.discover.save ? [saveSearch] : [],
+      ...(uiCapabilities.discover.save ? [saveSearch] : []),
```

Comment:
Why did the code mod choose to add brackets specifically to this line?

Reply:
There were some minor differences in how the code was output. I caught most of them but it looks like that one slipped by.